src/containers/edit-profile/profile-tab/profile-tab-form/ProfileTabForm.tsx

```diff
@@ -141,13 +142,9 @@ const ProfileTabForm: FC<ProfileTabFormProps> = ({
               validationData={validationData}
               variant={ButtonVariantEnum.ContainedLight}
             />
-            <AppButton
-              onClick={handleRemovePhoto}
-              size={SizeEnum.Large}
-              variant={ButtonVariantEnum.Tonal}
-            >
+            <Button onClick={handleRemovePhoto} size='md' variant='tonal'>
```

Comment:
in a lot of places you changed the sizes, for example here it was `large` and you changed to `medium`. does this align with the design?

Reply:
yeah, it`s more corresponding to the design . As xxl was removed md its bigger than our previous md